lib/msal-core/samples/react-sample-app/src/AuthProvider.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from "react";
+import * as Msal from "msal";
+
+const loginRequest = {
+    scopes: ["openid", "profile", "User.Read"]
+};
+
+const tokenRequest = {
+    scopes: ["Mail.Read"]
+};
+
+const graphConfig = {
+    graphMeEndpoint: "https://graph.microsoft.com/v1.0/me",
+    graphMailEndpoint: "https://graph.microsoft.com/v1.0/me/messages"
+};
+
+const msalApp = new Msal.UserAgentApplication({
+    auth: {
+        clientId: "245e9392-c666-4d51-8f8a-bfd9e55b2456",
+        authority: "https://login.microsoftonline.com/common",
+        validateAuthority: true,
+        postLogoutRedirectUri: "http://localhost:3000"
+    },
+    cache: {
+        cacheLocation: "localStorage",
+        storeAuthStateInCookie: true
```

Comment:
Have the 'readme' add that this is mandatory only for IE/Edge. Essentially for any browser that has trusted zones enabled and the browser storage is cleared on redirect. Making this a part of the sample may generalize it for all the users which we are not supposed to.

Reply:
Should we add as a comment in the code instead / as well?